exporters/zipkin/zipkin_test.go

```diff
@@ -31,6 +32,7 @@ import (
 	zkmodel "github.com/openzipkin/zipkin-go/model"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/tonglil/buflogr"
```

Comment:
Can we use https://pkg.go.dev/github.com/go-logr/logr/funcr#New instead so that we do not introduce a new dependent Go module?